### ISSUE TITLE
cssnext already have a "nesting" feature

### ISSUE THREAD

**Issue by MoOx** [jongleberry/postcss-jongleberry#17]
Not sure you where aware of that (I saw postcss-nested)

http://cssnext.io/features/#nesting

**Comment by jonathanong** [comment]
oh shoot. that's on track to be a standard? is it using the same module?

**Comment by MoOx** [comment]
Yes it's going to be a standard. Not the same module as there is some small differences https://github.com/jonathantneal/postcss-nesting

**Comment by jonathanong** [comment]
fuck `&` is required everywhere.

**Comment by MoOx** [comment]
that's the spec :/
